### DISCUSSION TITLE
devices roles, Device Type and other added item not shown in other admin IP

### DISCUSSION THREAD

**Discussion by alirezaunix** [netbox-community/netbox#9157] [Q&A]
### NetBox version

3.1.6

### Python version

3.8

### Steps to Reproduce

in other IP with the same user
in new devices
no device type or device role shown

### Expected Behavior

I except to show the added item like my own PC 

### Observed Behavior

but nothings show in the drop list

**Comment by ghmarcon** [comment]
Error: Invalid or missing CSRF token

stack: "Error: Invalid or missing CSRF token\n at yI (https://netbox.localhost/static/netbox.js?v=3.2.2:8:
